tests/rpc/test_pythongen.ml

```diff
@@ -105,8 +106,13 @@ let check_test_class () =
   run "Calc.noop false" |> ignore;
   run ~input:{|{"bool1":true}|} "Calc.noop --json" |> ignore
 
+let check_exceptions () =
+  gen_python "python/bindings.py";
```

Comment:
should this be `exn_test.py`?

Reply:
No, this is the filename of the generated bindings, `exn_test.py` uses that.

Reply:
I've renamed the function to make it clearer what it does.